dev/specs/pub_pub_video_call.yml

```diff
@@ -0,0 +1,34 @@
+kind: Room
+id: pub-pub-video-call
```

Comment:
Use hyphens `-` for file names if your `id` use them. No need to use `_` in file names and provide another indirection level.